app/steps/ui/summary/template.html

```diff
@@ -14,36 +15,49 @@ <h1 class="heading-large">
     </h1>
     <p>{{content.Summary.checkCarefully}}</p>
 
-    <h2 class="heading-medium">
-        {{content.Summary.willWithCodicilHeading if fields.summary.codicilPresent.value === "true" else content.Summary.willHeading}}
-    </h2>
-    <dl class="check-your-answers check-your-answers--long">
-        {% include "ui/summary/includes/will.njk" %}
-    </dl>
-
-    <h2 class="heading-medium">{{content.Summary.ihtHeading}}</h2>
-    <dl class="check-your-answers check-your-answers--long">
-        {% include "ui/summary/includes/iht.njk" %}
-    </dl>
-
-    <h2 class="heading-medium">{{content.Summary.applicantHeading}}</h2>
-    <dl class="check-your-answers check-your-answers--long">
-        {% include "ui/summary/includes/executors.njk" %}
-    </dl>
-
-    <h2 class="heading-medium">{{content.Summary.deceasedHeading}}</h2>
-    <dl class="check-your-answers check-your-answers--long">
-        {% include "ui/summary/includes/deceased.njk" %}
-    </dl>
+    {% ifAsync fields.summary.isScreeningQuestionToggleEnabled.value === 'true' %}
```

Comment:
Did we decide that it was tricky component testing the `ifAsync` code?

Reply:
Don't think we decided anything...

Reply:
In that case, can we add component tests for this with toggle enabled and disabled?

Reply:
Added